packages/vite/src/__tests__/main.test.ts

```diff
@@ -7,7 +7,15 @@ import { OutputAsset, OutputChunk, RollupOutput } from 'rollup'
 import { JSDOM } from 'jsdom'
 import sharp from 'sharp'
 import { afterEach, describe, test, expect, it, vi } from 'vitest'
-import { createBasePath } from '../utils'
+import { createBasePath, generateCacheID } from '../utils'
+import { readFile, rm, utimes } from 'fs/promises'
+
+const processPath = process.cwd()
```

Comment:
I think `cwd` would be a clearer name. I know we used that variable name in the sveltekit codebase in the path so it would help keep them aligned
```suggestion
const cwd = process.cwd()
```

Reply:
Changed